test/e2e/config/rabbitmqvhost/rabbitmqvhost.go

```diff
@@ -18,19 +18,65 @@ package rabbitmqvhost
 
 import (
 	"context"
+	"embed"
+	"log"
 
-	"knative.dev/reconciler-test/pkg/environment"
+	"github.com/kelseyhightower/envconfig"
```

Comment:
@gabo1208 are we using this library somewhere else?

Reply:
Yes in all of the mains that load somethkng from the env, the dispatcher, adapter, controllers, etc... why?